service/src/test/java/com/commercetools/pspadapter/payone/notification/common/CaptureNotificationProcessorTest.java

```diff
@@ -113,22 +119,26 @@ public void processingPendingNotificationForPendingChargeTransactionDoesNotChang
 
     @Test
     @SuppressWarnings("unchecked")
-    public void processingCompletedNotificationForPendingChargeTransactionDoesNotChangeState() throws Exception {
-        final Payment payment = processingCompletedNotificationForPendingChargeTransactionDoesNotChangeStateWireframe();
+    public void processingCompletedNotificationForPendingChargeTransactionChangesStateToSuccess() throws Exception {
+        final Payment payment = processingCompletedNotificationForPendingChargeTransactionChangesStateWireframe();
         verifyUpdateOrderActions(payment, ORDER_PAYMENT_STATE);
```

Comment:
Maybe you can use this naming convention to make the testname more understandable:
MethodName_StateUnderTest_ExpectedBehavior

Reply:
maybe in the future - it requires re-factoring hundreds of functions